response_operations_ui/common/mappers.py

```diff
@@ -22,7 +23,9 @@ def convert_events_to_new_format(events):
         formatted_events[event['tag']] = {
             "day": day,
             "date": date,
-            "time": time
+            "month": str(date_time.month) if len(str(date_time.month)) == 2 else f"0{date_time.month}",
+            "time": time,
+            "is_in_future": date_time > iso8601.parse_date(datetime.now().isoformat())
```

Comment:
Why do you need to parse the datetime.now() to compare with another datetime object? Shouldn't date_time > datetime.now() work?

Reply:
I'll check

Reply:
We do need to cast the date with iso8601. When we parse the initial date it converts the `+00:00` into timezone information. You cant compare a date with a given timezone to one without so we also cast the `now` datetime like this